components/speedreader/speedreader_util.cc

```diff
@@ -62,4 +72,49 @@ bool IsSpeedreaderPanelV2Enabled() {
   return base::FeatureList::IsEnabled(speedreader::kSpeedreaderPanelV2);
 }
 
+void DistillPage(const GURL& url,
+                 std::string body,
+                 SpeedreaderService* speedreader_service,
+                 SpeedreaderRewriterService* rewriter_service,
+                 DistillationResultCallback callback) {
+  struct Result {
+    DistillationResult result;
+    std::string body;
+    std::string transformed;
+  };
+
+  auto distill = [](const GURL& url, std::string data,
+                    std::unique_ptr<Rewriter> rewriter) -> Result {
+    SCOPED_UMA_HISTOGRAM_TIMER("Brave.Speedreader.Distill");
+    int written = rewriter->Write(data.c_str(), data.length());
+    // Error occurred
+    if (written != 0) {
+      return {DistillationResult::kFail, std::move(data), std::string()};
+    }
+
+    rewriter->End();
+    const std::string& transformed = rewriter->GetOutput();
+
+    if (transformed.length() < 1024) {
```

Comment:
i think there was a comment explaining this `if`

Reply:
You're wrong 
![image](https://user-images.githubusercontent.com/73575789/226517409-a1862c70-55c1-4580-b5c5-7b93f7ca0039.png)

Reply:
![image](https://user-images.githubusercontent.com/73575789/226517563-e8d2d7ec-5d3c-497a-bfdc-146a437da850.png)
Useless reference to the closed issue